### PR TITLE
#258 - Add radio buttons

### LINKED ISSUE #258: Add radio buttons
Change these checkbox selections to radio buttons, since only one should be allowed to be selected:
image
image

### PR DESCRIPTION
**Pull request by amedenblik**
Closes #258

This should fix the errors we were getting with https://github.com/USGS-WiM/StreamStats-National/pull/249.

### CODE REVIEW

**Comment by amedenblik** [comment]
Converted to draft because I realized you can select more than 1 radio button: 
image

**Comment by amedenblik** [comment]
Only one radio button can now be selected. Ready for review!

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Remove all output workflow layers when workflow is complete
 - Updated to google analytics 4
 - Burn Severity is turned off by default for Fire Hydrology workflows
+- Some checkboxes are now radio buttons where appropriate
+- Default option selected value from workflow.json now works correctly
```
Comment (harper-wavra) [inline CHANGELOG.md:30]: I am not seeing any default options selected

**Review by harper-wavra — COMMENTED** [review]
Works well! I just wasn't seeing any default options, but maybe we don't need any? Then we can just remove that line from the changelog.

**Review by harper-wavra — APPROVED** [review]

**Pull request merged**